pkg/odo/cli/application/application.go

```diff
@@ -350,11 +408,11 @@ func printDeleteAppInfo(client *occlient.Client, appName string) error {
 	}
 
 	for _, currentComponent := range componentList {
-		componentDesc, err := component.GetComponentDesc(client, currentComponent.Name, appName)
+		componentDesc, err := component.GetComponentDesc(client, currentComponent.ComponentName, appName)
 		if err != nil {
 			return errors.Wrap(err, "unable to get component description")
 		}
-		log.Info("Component", currentComponent.Name, "will be deleted.")
+		log.Info("Component", currentComponent.ComponentName, "will be deleted.")
```

Comment:
Now since its already <...>component.Name I tend to feel Name is better off than ComponentName

Reply:
oh I will do it, it is there because, I removed `Info` struct and reused `Description`